app/views/admin/accounting/transactions/_transactions_grid_definition.html.slim

```diff
@@ -10,3 +10,20 @@
   - g.column name: t('activerecord.attributes.accounting.transaction.amount'),
     filter: false do |transaction|
     - format_currency transaction.amount, transaction.currency
+  - g.column name: t('activerecord.attributes.accounting.transaction.interest_paid'),
+    filter: false do |transaction|
+    - if transaction.loan_transaction_type == 'repayment'
+      - format_currency transaction.change_in_interest, transaction.currency
```

Comment:
These are potentially good candidates for decorators in the future -- might be worth suggesting when the project people get back.